.sync.yml

```diff
@@ -18,7 +18,7 @@
   includes:
   - bundler_args: 
     dist: trusty
-    env: PLATFORMS=debian_puppet5
+    env: PLATFORMS=deb_puppet5
     rvm: 2.5.3
     before_script:
     - bundle exec rake 'litmus:provision_list[waffle_deb]'
```

Comment:
Hey @david22swan ,

This should be updated to travis_deb.

Cheers!

Reply:
fixed